package.nls.zh-tw.json

```
@@ -13,13 +14,13 @@
     "python.command.python.runLinting.title": "執行 Linting",
     "python.snippet.launch.standard.label": "Python: 目前檔案",
     "python.snippet.launch.module.label": "Python: 模組",
-    "python.snippet.launch.module.default": "請輸入模組名稱",
+    "python.snippet.launch.module.default": "請輸入-模組-名稱",
     "python.snippet.launch.django.label": "Python: Django",
     "python.snippet.launch.fastapi.label": "Python: FastAPI",
     "python.snippet.launch.flask.label": "Python: Flask",
     "python.snippet.launch.pyramid.label": "Python: Pyramid 程式",
     "python.snippet.launch.attach.label": "Python: 附加",
-    "python.snippet.launch.attachpid.label": "Python: 使用 PID 附加",
+    "python.snippet.launch.attachpid.label": "Python: 使用處理程序 ID 附加",
```

```suggestion
    "python.snippet.launch.attachpid.label": ""Python: 使用處理程 ID 附加"",
```
This appears to be more accurate as per google translation, does this look fine?

---

No. "處理程" isn't a valid word in Chinese.
In Wikipedia, "行程識別碼" is corresponding to PID, but the word isn't more common than PID for Chinese users.
I think the translation of "python.snippet.launch.attachpid.label" can keep 
the original translation "Python: 使用 PID 附加".

---

I see, alright you can revert your change then.

---

```suggestion
    "python.snippet.launch.attachpid.label": "Python: 使用 PID 附加",
```